sql_metadata/token.py

```diff
@@ -217,7 +250,7 @@ def table_prefixed_column(self, table_aliases: Dict) -> str:
         value = self.left_expanded
         if "." in value:
             parts = value.split(".")
-            if len(parts) > 3:  # pragma: no cover
+            if len(parts) > 2:  # pragma: no cover
```

Comment:
Did we stop supporting `database.table.column` notation?

Reply:
When i checked some backends it appeared it's not possible. You can have 3 part table names but not column names. If you have some backend that supports it we can revert.

Reply:
But then we don't have any test for it :)

Reply:
Makes sense, thanks for clarifying 🙂